### PR TITLE
[zh] Update concepts/security

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.
The docs of content/zh/docs/concepts/security/index.md has been outofsync, please see where the English documentation is updated :
image

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/concepts/security/index.md [hunk 1]
```diff
@@ -651,6 +651,18 @@ Istio 使用双向 TLS 将某些信息从客户端安全地传递到服务器。
...
 - `source.namespace` 自定义条件
-- `connection.sni` 自定义条件
 
-如果您不使用授权策略中的上述任何字段，则双向 TLS 不是必须的。
+请注意，强烈建议始终在 PeerAuthentication 中将这些字段与严格的双向 TLS 模式一起使用，
+以避免在将纯文本流量与允许的双向 TLS 模式一起使用时潜在的意外请求拒绝或绕过安全策略。
+
+如果您无法启用严格的双向 TLS 模式，请查看[安全公告](/zh/news/security/istio-security-2021-004)以获取更多详细信息和替代方案。
+
+## 学习更多
```
Comment (kebe7jun) [inline content/zh/docs/concepts/security/index.md:660]: 需要加 id。

File: content/zh/docs/concepts/security/index.md [hunk 2]
```diff
@@ -651,6 +651,18 @@ Istio 使用双向 TLS 将某些信息从客户端安全地传递到服务器。
...
+
+在学习了基本概念之后，还有更多的资源可以回顾：
```
Comment (windsonsea) [inline content/zh/docs/concepts/security/index.md:662]:
```suggestion
在学习了上述基本概念之后，您还可以温习下述资料：
```

File: content/zh/docs/concepts/security/index.md [hunk 3]
```diff
@@ -651,6 +651,18 @@ Istio 使用双向 TLS 将某些信息从客户端安全地传递到服务器。
...
+
+- 按照[身份验证](/zh/docs/tasks/security/authentication/authn-policy)和[授权](/zh/docs/tasks/security/authorization)任务尝试使用安全策略。
+
+- 了解一些可用于提高网格安全性的安全[策略示例](/zh/docs/ops/configuration/security/security-policy-examples)。
+
+- 阅读 [常见问题](/zh/docs/ops/common-problems/security-issues/)，以便在出现问题时更好地解决安全策略问题。
```
Comment (windsonsea) [inline content/zh/docs/concepts/security/index.md:668]:
```suggestion
- 阅读[常见问题](/zh/docs/ops/common-problems/security-issues/)，以便在出现问题时更好地解决安全策略问题。
```

File: content/zh/docs/concepts/security/index.md [hunk 4]
```diff
@@ -651,6 +651,18 @@ Istio 使用双向 TLS 将某些信息从客户端安全地传递到服务器。
 - `source` 部分下的 `namespaces` 字段
 - `source.principal` 自定义条件
 - `source.namespace` 自定义条件
-- `connection.sni` 自定义条件
 
-如果您不使用授权策略中的上述任何字段，则双向 TLS 不是必须的。
+请注意，强烈建议始终在 PeerAuthentication 中将这些字段与严格的双向 TLS 模式一起使用，
+以避免在将纯文本流量与允许的双向 TLS 模式一起使用时潜在的意外请求拒绝或绕过安全策略。
```
Comment (windsonsea) [inline content/zh/docs/concepts/security/index.md:656]:
```suggestion
请注意，强烈建议始终在 PeerAuthentication 中使用这些字段开启 STRICT 模式的双向 TLS，
以避免在 PERMISSIVE 模式的双向 TLS 中使用纯文本流量时出现意外的请求拒绝或绕过安全策略。
```

**Review by windsonsea — APPROVED** [review]

**Pull request merged**